src/app/about/about.page.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { NetworkService } from '../network.service';
-import { AppConfigs } from '../app.config';
+import { AppConfigs } from '../core-module/constants/api.config';
```

Comment:
File name should be app.config, not api.config.

Reply:
changed